shuup/simple_supplier/module.py

```diff
@@ -56,8 +56,10 @@ def update_stock(self, product_id):
         sv, _ = StockCount.objects.get_or_create(supplier_id=supplier_id, product_id=product_id)
         sv.logical_count = values["logical_count"]
         sv.physical_count = values["physical_count"]
-        latest_event = StockAdjustment.objects.filter(
-            supplier=supplier_id, product=product_id).order_by("-created_on").first()
+        latest_event = (
+            StockAdjustment.objects
+            .filter(supplier=supplier_id, product=product_id, type=StockAdjustmentType.INVENTORY)
+            .order_by("-created_on").first())
```

Comment:
is `.order_by("-created_on").first()` actually same as `.last()` in this queryset?